lib/generate-layouts.js

```diff
@@ -0,0 +1,223 @@
+"use strict";
+var _ = require("lodash");
+var fs = require("fs");
+/* eslint-disable no-magic-numbers */
+
+// Each layout consists of vertical panels, that contains its position and horizontal views.
+// Flex-like positions of panels and views defined by 'grow' and 'size' parameters.
+// View or panel with 'size' has exactly <size> height or width respectively.
+// View or panel with 'grow' fills <grow> part of the residuary space (it works like flex-grow).
+// By default, position = { grow: 1 }
+
+var DEFAULT_LAYOUT_CONFIG = [
```

Comment:
You are right, js is better here. I replaced `fs` with `require` - not completely sure if I require there in a right way, but it seems to work.